lib/gitarro/backend.rb

```diff
@@ -83,13 +114,12 @@ def open_newer_prs
     prs
   end
 
-  # public for etrigger the test
+  # public for retrigger the test
   def retrigger_check(pr)
     return unless retrigger_needed?(pr)
     create_status(pr, 'pending')
     exit 1 if @check
-    launch_test_and_setup_status(pr)
-    j_status == 'success' ? exit(0) : exit(1)
+    launch_test_and_setup_status(pr) == 'success' ? exit(0) : exit(1)
```

Comment:
For the future I really think we should move all exits to gitarro.rb

Reply:
moving the exit in the `bin` cmdline application, can cause that we need to expose maybe other functions to public for exiting, and make some other checks.

I can agree that having all exit in one place ( the commandline.rb) it is better.